services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/resource_tracker.py

```diff
@@ -23,8 +25,12 @@ async def _startup() -> None:
         if not settings:
             _logger.warning("Prometheus API client is de-activated in the settings")
             return
+        lock_key = f"{app.title}:collect_container_resource_usage_task_lock"
```

Comment:
this is probably ok for now. but for a later timepoint in case you need to have several of this service running in parallel for different services, then the lock key should have a value that changes with the service names you are watching for. Otherwise they will lock each other.
e.g. ```f"{app.title}:collect_container_resource_usage_task_lock_{service_watched_regex}"```

Reply:
you are right :+1: thanks